### PR TITLE
TE 7001 Symfony 5 support

### PR DESCRIPTION
**Pull request by stereomon**
- Developer(s): @stereomon

- Ticket: https://spryker.atlassian.net/browse/TE-7003

- Academy PR: ACADEMY_URL_HERE

- Release Group: https://release.spryker.com/release-groups/view/2897

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   Silexphp              | patch (currently 0.x)  |                       |

-----------------------------------------

##### Change log

Initial Release

- Allow installation of Symfony components v5.
- Replaced old Event classes with new ones.

### CODE REVIEW

**Comment by stereomon** [comment]
Previous PR https://github.com/spryker/silexphp/pull/33

**Review by a-sabaa — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -16,33 +16,36 @@
     ],
     "require": {
         "php": ">=7.2",
-        "spryker/container": "^1.1.0",
-        "symfony/event-dispatcher": "^3.0.0 || ^4.0.0",
-        "symfony/http-kernel": "^3.4.0 || ^4.0.0",
-        "symfony/routing": "^3.0.0 || ^4.0.0"
+        "spryker/container": "dev-beta/te-7001/te-7174/te-7175-create-beta-branch-for-spryker-container as 1.1.0",
```
Comment (dereuromark) [inline composer.json:19]: We need to revert this before tagging the release.

**Comment by dereuromark** [comment]
Whats the status of spryker/container ?
We cannot put this into release before this is resolved.

**Comment by stereomon** [comment]
The change was only needed in the beginning and there is no need to update the container for silex, I reverted it.

**Pull request merged**